etc/kayobe/ansible/vault-deploy-barbican.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Configure AppRole
+  any_errors_fatal: True
+  gather_facts: True
+  hosts: controllers[0]
+  vars:
+    vault_api_addr: "https://{{ kolla_internal_fqdn }}:8200"
+    vault_ca_cert: "{{ '/etc/pki/tls/certs/ca-bundle.crt' if ansible_facts.os_family == 'RedHat' else '/usr/local/share/ca-certificates/OS-TLS-ROOT.crt' }}"
+  tasks:
+    - name: Include Vault keys
+      include_vars:
+        file: "{{ kayobe_env_config_path }}/vault/overcloud-vault-keys.json"
+        name: vault_keys
+
+    - name: Ensure hvac is installed
```

Comment:
Missing upper constraints used elsewhere. Perhaps we should assume this is in place here?

Reply:
Done.